docs/importing_models.md

```diff
@@ -0,0 +1,16 @@
+# Introduction {#openvino_docs_importing_models_introduction}
+
+This chapter provides you with the information about available capabilities to get you model imported to OpenVINO™.
+
+Before starting inference with the OpenVINO™ toolkit, a model should be pre-trained in one of the AI frameworks and exported into one of the supported model formats. Many models represented in ONNX, TensorFlow, PaddlePaddle, MXnet, Caffe, and Kaldi formats are supported in OpenVINO. Dedicated APIs and command-line tools are used to import such models to OpenVINO. 
+
+The process of importing is also referred to as model conversion because the original model is being converted to the model format used in OpenVINO. Converted model can be either:
+
+*	Stored in the OpenVINO intermediate representation (IR) format: a pair of .xml and .bin files used for topology description and trained parameters respectively.
+*	Used for inference immediately in runtime after conversion. 
+
+To facilitate conversion to the IR file format, the [Model Optimizer](MO_DG/Deep_Learning_Model_Optimizer_DevGuide.md) tool is used. It reads the original model represented in one of the supported formats and produces IR files (.xml and .bin files). The main purpose of Model Optimizer is to prepare the model in an appropriate format for faster inference offline without spending time on model file conversion in the final inference application.
```

Comment:
Add `Model Optimizer also can optionally adjust the model to be more suitable for inference, for example alternating input shapes, embedding preprocessing and cutting training parts off.`